### DISCUSSION TITLE
Not getting verified despite of multiple attempts and all kinds of proofs.

### DISCUSSION THREAD

**Discussion by pnpn111** [community/community#51579] [GitHub Education] [answered]
**Since my English is not very good, the full text will be translated by machine. Please forgive me if there are some problems. Thank you!**

I tried applying for the Github Student Package several times and provided a range of materials. This information includes: 

1. Student ID with date and name, photo, school name and school seal (clear picture and English translation provided). 
2. Admission notice with the English name of the school. 
3. The website certificate of the contest I participated in at school (including the contest time, name and school, and the information and student ID photo should be taken together) is from the official organization of China Computer Society, which ensures its authority and does not need to worry about the possibility of tampering with the documents. 

But I still didn't succeed.

Here I will describe the reasons for rejecting the email and my attempts:

These reasons are based on multiple emails, not a single email, and I tried to avoid these problems in my last application.

**However, after all these attempts I still failed the audit**

- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
Therefore, I took photos and uploaded a number of pictures with translations or English annotations, including the translation of the student ID card and the admission notice with the English name of the school.

- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
This is the only time that I tried to upload picture files. Except this time, all other pictures were uploaded by taking photos. Meanwhile, I also tried to take photos with mobile phone and computer camera.

- Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.
The photos I uploaded all contain the English name of the school (or the translation of the school name) and the relevant date. After receiving this message, I re-specified the specific and clear shooting date and translated the date into English.

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
&
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?
These two problems always come up together, so I've also put them together and described my solution.I fully uploaded my information, including my real name, profile picture, profile and so on, and attached relevant links to prove my identity (but I have temporarily removed these information due to privacy reasons, but I submit the application properly so that I can guarantee that I have provided true and complete information).I can assure you that I have filled in all the details of my personal information (including my profile picture, etc.) when submitting the application.

Despite all the above attempts, I still failed to pass the application. Is there anything else I should do?Can someone give me some advice?Thanks!

**Comment by pnpn111** [comment]
Supplementary note: When submitting the application, I have tried unsuccessfully to use my school computer, my home computer, and my two mobile phones at home, all of which are in the same city as my school.I also didn't use a proxy to visit GitHub during the application submission process.

**Comment by pnpn111** [comment]
Was there something I didn't do or didn't do right? Can someone give me a reminder?

**Comment by samus-aran** [comment, marked as answer]
Hi @pnpn111 

Let's see how we can assist ⭐ 

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by pnpn111** [reply]
Thanks for your reply! I have tried to translate part of the content into English and improve my personal information. On the next attempt, I will try to provide a full translation and try again to refine the personal information.

In addition, would it be helpful if I hand-write the photo in the photo uploaded (for example, "photo was used for Github student package registration, user name pnpn111")?

**Reply by pnpn111** [reply]
Thank you very much for your help. With your reminding, I have corrected the error and successfully registered the Github student package.

**Comment by zeinabalalfy** [comment]
I have the exact same problem I put my university email the domain is correct and put my  academic affiliation but it is not in English
